### PR TITLE
Click logo to clear all input

### LINKED ISSUE #170: Bug(s) 🐛 🐞 🐜
Image

- [ ] Clear _input button_ function for the _text to be summarised_

### LINKED ISSUE #201: (Manual) E2E Testing

### PR DESCRIPTION
**Pull request by Rgonzales4**
Issue: https://github.com/SIS-Team-24/2023-SIS-Team-24/issues/201#issuecomment-1780463989

When all values and input are shown:
image

*clicks logo* aftermath:
image

### CODE REVIEW

**Review by gracebilliris — APPROVED** [review]
Looks good!

**Review by stephenb-coding — APPROVED** [review]
Very detailed description 👏 Looks good  💯

**Review by JamesBug404 — APPROVED** [review]
LGTM! Nice job.

File: client/src/pages/Home.tsx [hunk 1]
```diff
@@ -359,10 +359,24 @@ function Home(this: any) {
...
+    setFinalSummary("");
+    setInputValue("");
+    setSentimentText("");
+    setEmotionLabel("");
+    setSentimentScore(0);
+    setKeywords({});
+    setSentimentPlaceholder("");
+    setEmotionalPlaceholder("");
+    setWordCount(0);
+    setSelectedFont(null);
```
Comment (henrygoodman) [inline client/src/pages/Home.tsx:378]: Just wondering, do we want font to be reset?

All the other state makes sense, but IMO font choice should persist

Comment (Rgonzales4) [reply client/src/pages/Home.tsx:378]: Grace's use case was for everything to revert back - so I took into account the font. Another thing i just thought of was the way the font option is presented is similar to every other feature so it holds no place above or below to be treated differently 🤷 .

**Pull request merged**